### ISSUE TITLE
XY Offset Range

### ISSUE THREAD

**Issue by jawaff** [x40-Community/X40_PRO-Firmware#1]
Hey,

I was calibrating things and realized that the XY offset range in the 1.2.5.9 branch isn't working for me. Maybe that's because I'm on the x40v1, but my X offset needs to go to 352.93. I'm thinking about looking around for the range and rebuilding, but I figured I'd make a note of it.

**Comment by jawaff** [comment]
For others, the middle value for the range is defined here.

https://github.com/x40-Community/X40_PRO-Firmware/blob/94107d0fd8c5fa15a84399713ec8c7f4ea515005/Marlin/Configuration.h#L156

**Comment by x40-Community** [comment]
Changing it would result in an bug and not fix one. Please read the instructions!
Support only for X40-Community member.

**Comment by jawaff** [comment]
I saw your 5 offset comment, but the issue still exists and I cannot align the two extruders. I've followed the calibration instructions. The highest possible offset is the closest to the offset I need, while the lowest possible offset makes the alignment much worse. 352.92 is the offset that I need according to the calculations. I've gone through quite a bit of 2x2 square tests.

**Comment by x40-Community** [comment]
Yes, and everyone else doesn't have the problem, so it's not a bug, it's user error.

**Comment by jawaff** [comment]
There's plenty of other errors in the x40-community tools/docs that haven't been reported. I still think they're valuable, but errors happen especially when you don't have many people testing/reading. I've got my customers coming to me with user errors and sometimes real errors all the time, so I know how that is. I did many tests to make sure I was doing it right, but I can try again. Maybe the 2x2 square file loading in with the wrong extruders according to the instructions tripped me up, even though I accounted for it in the calculation.

**Comment by x40-Community** [comment]
The software was tested over two years and only improved in small steps. Errors have not been reported in the last few months and I haven't found any either.
If you know how to do it, then it works without errors and immediately. Most X40 community members are very happy with the solution. Can you go ask on the forums. Simply use the Weedo Stockfirmware move the left extruder to the park position and the right one to X=0, then you will certainly understand the bugfix from the X40 PRO Firmware.

**Comment by jawaff** [comment]
"It works on my machine" isn't always a valid excuse, but I'll double check
the calibration tests.
